packages/node_modules/pouchdb-adapter-http/src/index.js

```diff
@@ -296,6 +296,10 @@ function HttpPouch(opts, callback) {
         /* istanbul ignore next */
         params.attachments = true;
       }
+      if (opts.latest) {
+        /* istanbul ignore next */
+        params.latest = true;
```

Comment:
Any chance we can have a test that hits this so we dont need to add an istanbul ignore?